### PR TITLE
fix key warning in widget

### PR DESCRIPTION
**Pull request by songgao**
filename from the same TLF can be same as they can be in different subdirs

### CODE REVIEW

File: shared/menubar/files.desktop.js [hunk 1]
```diff
@@ -98,9 +98,11 @@ const defaultNumFileOptionsShown = 3
 
 const FileUpdates = (props: FileUpdatesProps & FileUpdatesHocProps) => (
   <Kb.Box2 direction="vertical" fullWidth={true}>
-    {props.updates.slice(0, props.isShowingAll ? props.updates.length : defaultNumFileOptionsShown).map(u => (
-      <FileUpdate key={u.name} {...u} />
-    ))}
+    {props.updates
+      .slice(0, props.isShowingAll ? props.updates.length : defaultNumFileOptionsShown)
+      .map((u, index) => (
+        <FileUpdate key={String(index)} {...u} />
```
Comment (jzila) [inline shared/menubar/files.desktop.js]: Shouldn't this be globally unique for a given folder? i.e. something like the path?

Comment (songgao) [reply shared/menubar/files.desktop.js]: That's not too bad; it'll just re-use the components and update everything when things shift around. These rows are pretty simple so I don't think it's gonna be a problem. We actually use this pattern elsewhere, and even have `indexAsKey` in `Kb.List`.

I guess using `path` would be better, but it isn't available here, so we'd need to do some plumbing to make it that way.

Comment (chrisnojima) [reply shared/menubar/files.desktop.js]: All the key does is help resolve if children move around. Aka if you insert an item at the front and items all moved back an index react could not  rerender them. Else itd rerender them all. These things should be connected so they'd rerender if their own data changed

Comment (jzila) [reply shared/menubar/files.desktop.js]: Right but that's the problem--a new item at the beginning wouldn't be rerendered if you used the index as a key. Or am I missing something?

**Comment by songgao** [comment]
@jzila please take another look; thanks!

**Pull request merged**